tests/fdb/test_fdb.py

```diff
@@ -318,7 +328,7 @@ def test_fdb(ansible_adhoc, ptfadapter, duthosts, rand_one_dut_hostname, ptfhost
     total_mac_count = 0
     for k, v in fdb_fact.items():
         assert v['port'] in interface_table
-        assert v['vlan'] in interface_table[ifname]
```

Comment:
>ifname [](http://example.com/codeflow?start=44&length=6)

Seems a bug which could be caught by static linter. Could you explore why it is not caught at first place?